ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -283,6 +283,42 @@ def test_paddle__reshape(
     )
 
 
+# backward
+
+
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="backward",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_paddle_backward(
+    dtype_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    on_device,
+    backend_fw,
+):
+    input_dtype, x = dtype_and_x
+    x = Tensor(x)
+    helpers.test_frontend_method(
```

Comment:
`x = Tensor(x)` should not be needed here. I believe this is done inside `test_frontend_method` later on.

Reply:
@AnnaTz
 I passed the five tests only after I add  x = Tensor(x), so I think it’s necessary